src/chrome/content/rules/Madison-Logic.xml

```diff
@@ -66,15 +58,12 @@
 	<securecookie host="^\.madisonlogic\.com$" name="^__(?:utm\w|zlcid)$" />
```

Comment:
Since there is no working URL known for the working targets, would you please remove the securecookie tags?

Reply:
With the leading `.`, that looks like a domain cookie, it which case there are still working targets if as you asked for clk, jsc and st to be enabled.

Reply:
> With the leading ., that looks like a domain cookie

This is a common confusion on the implementation of the `securecookie` tag. In fact, HTTPSE is only capable of protecting cookies set for a domain in the list of `target` if a matching `securecookie` tag is found. If you would like to protect the cookies anyway, you can use a trivial securecookie rule `<securecookie host=".+" name=".+" />`

Reply:
Thanks for clearing that up.